lib/react_client/react_interop.dart

```diff
@@ -624,17 +586,17 @@ class InteropProps implements JsMap {
   @Deprecated('Only used with the deprecated Component base class and not Component2,'
       ' which stores props as properties on this object.')
   external ReactDartComponentInternal get internal;
-  external dynamic get key;
-  external dynamic get ref;
+  external dynamic /*?*/ get key;
+  external dynamic /*?*/ get ref;
```

Comment:
Should these be `String` / `Object` to match the `InteropProps` argument types?

Reply:
Mm good catch. Since that would be a breaking change, I think I'd rather just update the types of `InteropProps`'s `key/`ref`arguments (which I can't find any usages of FWIW) to `dynamic` instead. 

Or just leave them as-is 🤷.